docs/UserGuide.md

```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
 
 ---
 
-### Sorting the shows
-[TO CHANGE AFTER REWORK]
+### Sorting the shows `sort`
 
-Sort shows based on the input prefix.
+Your show list is in a mess! Do you need a way to organise your list?
+Sort can help reorder the list!!
 
-Format: `sort [sna/] [snd/] [ssa/] [ssd/] [so/]…​`
-* Use sna/ to sort name in ascending order.
-* Use snd/ to sort name in descending order.
-* Use ssa/ to sort status in ascending order.
-* Use ssd/ to sort status in descending order.
-* If there are no prefix, it will sort by name in ascending order.
-* If both prefixes for ascending and descending are used for the same attribute, it will only sort by ascending.
-* If both prefixes for name and status are used, by default, it will sort by name then status.
-* Use so/ to sort by status then name.
+:exclamation:**Caution:**<br>
+*You cannot undo the sort once you have applied it!
+
+Format: `sort [n/ORDER] [s/ORDER] [t/ORDER] [r/ORDER] [so/SEQUENCE]…​`
+
+`Sort` will sort according to the [prefixes](#command-structure)**
+* Use `n/` if you want to sort by name 
+* Use `s/` if you want to sort by status
+* Use `t/` if you want to sort by tag
+* Use `r/` if you want to sort by rating
+* For the above prefixes, you have to specify the `ORDER` right after the prefix.
+* `ORDER` is the order you want to sort the list by! Type `asc` to sort by ascending or type `dsc` to sort by descending! 
+* If two or more of the above prefixes are being used, sort will prioritise sorting by name then status then rating then tag.   
```

Comment:
Sorry, I think I missed this the last time I was reviewing
`sort will prioritise sorting by name, then status, followed by rating, and finally tags.`